pr.js

```diff
@@ -43,10 +43,24 @@ function getReleaseType(pr, config) {
 function getReleaseNotes(pr, config) {
     let notes = ''
     if (pr.body !== null && pr.body !== '') {
-        const matches = pr.body.match(config.releaseNotesRegex)
-        if (matches !== null && matches.length > 1) {
-            notes = matches[1].trim()
+        let lines = pr.body.split(/\r?\n/);
+        let withinNotes = config.releaseNotesPrefixPattern === undefined;
+
+        for (let i in lines) {
+            let line = lines[i];
+
+            if (withinNotes) {
+                if (config.releaseNotesSuffixPattern !== undefined && config.releaseNotesSuffixPattern.test(line)) {
+                    break;
+                }
+
+                notes += line + "\n";
```

Comment:
Rather than building up a new string like this (or copying lines into a new array, etc), it would be better to simply detect the indices of the lines matching the prefix and suffix, then simply call `.join(' ')` on a subrange of the original `lines` array.